test/test_manifest.json

```diff
@@ -3537,5 +3537,12 @@
        "md5": "ced0e2d88cfd5b4d3a55d937ea288af1",
        "rounds": 1,
        "type": "eq"
+    },
+    {  "id": "pr8491",
+       "file": "pdfs/pr8491.pdf.link",
```

Comment:
I don't think this will work, it needs to read `"file": "pdfs/pr8491.pdf"` instead.